src/app/shared/image/image-upload.component.spec.ts

```diff
@@ -12,6 +12,8 @@ describe('ImageUploadComponent', () => {
   });
 
   let imageDataURI = 'data:image/gif;base64,R0lGODlhAQABAIAAAAUEBAAAACwAAAAAAQABAAACAkQBADs=';
+  let notSquareImageDataURI = 'data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAAA4AAAAJCAYAAAACTR1pAA\
+AAGXRFWHRTb2Z0d2FyZQBBZG9iZSBJbWFnZVJlYWR5ccllPAAAABpJREFUeNpi/P//PwM5gImBTDCqcVBpBAgwAAtnAw/QRSp6AAAAAElFTkSuQmCC';
```

Comment:
I spent entirely too long trying to find a non square small enough image data uri that Phantom would agree with.